lib/matplotlib/cm.py

```diff
@@ -130,7 +130,7 @@ def __getitem__(self, item):
         try:
             return self._cmaps[item].copy()
         except KeyError:
-            raise KeyError(f"{item!r} is not a known colormap name")
+            raise KeyError(f"{item!r} is not a known colormap name") from None
```

Comment:
Should one possibly list all the possible ones here? As requested in https://github.com/matplotlib/matplotlib/issues/22637 (but not for colormap).

Otherwise makes sense!

Reply:
Although I now realize that the "working" example is for colormaps, but apparently triggered elsewhere.

Reply:
The list is extremely long and I'm not sure it would really improve legibility.  But in any case this can be discussed separately...